databricks/notebooks/databricks_mlflow_rapids.ipynb

```diff
@@ -0,0 +1,412 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import time\n",
+    "import subprocess\n",
+    "import sys\n",
+    "import threading\n",
+    "from queue import Queue, Empty\n",
+    "\n",
+    "from functools import partial\n",
+    "\n",
+    "import mlflow\n",
+    "import mlflow.sklearn\n",
+    "\n",
+    "from cuml.metrics.accuracy import accuracy_score as cuml_acc\n",
```

Comment:
Just for the sake of consistency between the two notebooks, we can avoid aliases. Use `cuml.ensemble.RandomForestClassifier`, `cuml.metrics.accuracy.accuracy_score`, etc., just so the user is clear that it comes from cuml.